### PR TITLE
[Feature] Add progress action

### PR DESCRIPTION
**Pull request by informalict**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +12,6 @@
 - (Bugfix) Change SyncWorker Affinity to Soft
 - (Feature) Add HostAliases for Sync
 - (Bugfix) Always stop Sync if disabled
-- (Bugfix) Fix checksum of accepted spec
```
Comment (ajanikow) [inline CHANGELOG.md:14]: Please, fix

File: pkg/apis/deployment/v1/plan.go [hunk 2]
```diff
@@ -97,6 +97,8 @@ type Action struct {
 	Locals PlanLocals `json:"locals,omitempty"`
 	// ID reference of the task involved in this action (if any)
 	TaskID types.UID `json:"taskID,omitempty"`
+	// Progress describes what is a status of the current action.
+	Progress string `json:"progress,omitempty"`
```
Comment (ajanikow) [inline pkg/apis/deployment/v1/plan.go:101]: Try to run update-generated target

**Review by ajanikow — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (informalict) [reply CHANGELOG.md]: fixed

File: pkg/apis/deployment/v1/plan.go (hunk 2 above)
Comment (informalict) [reply pkg/apis/deployment/v1/plan.go:101]: done

**Pull request merged**